packages/admin-frontend/components/Editor/index.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2018 GDL
+ *
+ * See LICENSE
+ */
+
+import React, { Fragment } from 'react';
+import dynamic from 'next/dynamic';
+import Link from 'next/link';
+
+import type { BookDetails, Chapter } from '../../types';
+import { saveBook, saveChapter } from '../../lib/fetch';
+
+// ReactJson doesn't work on the server, so make sure we lazy load it
+const ReactJson = dynamic(import('react-json-view'));
```

Comment:
Since this module is only used in the admin-frontend, remove the dependency in gdl-frontend